azurerm/internal/services/network/tests/network_watcher_flow_log_resource_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	nw "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network"
 )
 
-func testAccAzureRMNetworkWatcherFlowLog_basic(t *testing.T) {
+func TestAccAzureRMNetworkWatcherFlowLog_basic(t *testing.T) {
```

Comment:
these are intentionally a combined test since you can only provision one Network Watcher at a time - could we revert the changes in this file?

Reply:
I found that the test suite is a little bit too long. Changing the test cases into public permit us to directly test the specific case or test only `network_watcher_flow_log_resource_test.go` cases. We could just set `parallel=1` to limit the provision number in testing. Would you still like to revert them as origin?

Reply:
@yupwei68 - please revert this and use the shared test runner, and will cause CI/CD to fail. this will also prevent users from running into this issue not knowing that they cannot be run in parallel - if you personally want to just run that test you can comment the others out while working on it